internals/overlord/stateengine.go

```diff
@@ -92,18 +126,17 @@ func (se *StateEngine) Ensure() error {
 	se.mgrLock.Lock()
 	defer se.mgrLock.Unlock()
 	if se.stopped {
-		return fmt.Errorf("state engine already stopped")
+		return ErrStateEngineStopped
 	}
 	var errs []error
 	for _, m := range se.managers {
 		err := m.Ensure()
 		if err != nil {
-			logger.Noticef("state ensure error: %v", err)
```

Comment:
Kind of the opposite of above: I notice you removed the logging here -- why? I just want *ensure* the callers of Ensure always log/handle the error?